src/components/ScrollableContainer/styled.ts

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+
+export const ScrollableContainer = styled.div`
+  margin-top: 20px;
+  max-height: 50vh;
+  overflow-y: scroll;
```

Comment:
I recommend using auto instead of scroll, so the scroll bar is not visible when it is disabled.
```suggestion
  overflow-y: auto;
```